### ISSUE TITLE
Fix expansions (qe = que)

### ISSUE THREAD

**Issue by allisonmargaret** [Multepal/popolwuj#20]
We told students to mark them up as <choice>: https://docs.google.com/document/d/1GwbNFBRMB1ez3mKjkX1lcKWWId2W0kEmBZl3Nko7vt0/edit. 

But not everyone did. They display incorrectly on the published edition.

**Comment by aldobarriente** [comment]
I've added most abbreviations to the q’ character. Should I go further and add the expansions/choices for the abbreviations that don't have it?

**Comment by allisonmargaret** [comment]
I’d say yes so that the markup is consistent — and ready for our expansion edition.

Thanks, Aldo!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by aldobarriente** [comment]
A small question perhaps more for the future: is there any way to improve the readability of the expansion notation? As I'm going through, the code is becoming less and less readable. Maybe there's a way to collapse it in some form?

**Comment by allisonmargaret** [comment]
Hi Aldo,

Is it unreadable because there are so many abbreviations? Or because the "<choice> <abbr><expan>" markup that we assigned is cumbersome? (Here's how we told students to do markup on abbreviations: https://docs.google.com/document/d/1GwbNFBRMB1ez3mKjkX1lcKWWId2W0kEmBZl3Nko7vt0/edit#heading=h.kj6gi61vda6w. Jump to the section "Ximénez's shorthand.")

The idea of "<choice> <abbr><expan>" was that we'd have one version of the text in which users can turn off abbreviations to make it easier to read (of interest to a general audience), and one version where they can see the original abbreviations (of interest to specialists).

I have been fighting a bug for the past few days so I am not going to make it tomorrow. I will write up a list of the pros and cons of GitHub desktop vs. Code and send it to the group or post to the shared folder.

Thanks for all of your work on this,
Allison

Allison Margaret Bigelow
433 New Cabell Hall / PO Box 400777
Department of Spanish, Italian, and Portuguese
University of Virginia
Charlottesville, VA 22904
amb8fk@virginia.edu * +1.434.243.3371 (p)
http://spanitalport.as.virginia.edu/people/profile/amb8fk
________________________________

**Comment by aldobarriente** [comment]
I don't think we should change the choice of abbreviation and expansion notation! It just seemed like at certain points in the text when abbreviations are used on one line, it makes the line harder to read (but that's more of a code readability problem since it won't be an issue on the website). My line of thinking was more so if this
`<choice><abbr>q’</abbr><expan instant="false">que</expan></choice>`
could somehow be collapsed into something that could look like
`<choice>...</choice>`
but then that raises issues of not knowing what is being abbreviated to begin with.

**Comment by aldobarriente** [comment]
Another note: I just noticed that some of the choice notation has separated _porq'_ into _por_ and _q'_, appearing in code like this
`por<choice><abbr>q’</abbr><expan instant="false">que</expan></choice>`
Should I go back and add the _por_ back inside? It doesn't happen with all instances of _porque_, but it might be better to be consistent about them

**Comment by allisonmargaret** [comment]
Are they inconsistent because Ximénez wrote them differently or because the people doing markup interpreted them differently?

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by aldobarriente** [comment]
I hadn't checked the folios to verify, so I will do that as I am revisiting those markups. Should we only consider _porq'_ the abbreviation when there is no space between the _por_ and _q_ ?

**Comment by aldobarriente** [comment]
![image](https://user-images.githubusercontent.com/45946179/65881816-fac41c80-e361-11e9-976b-08afd79c5c5d.png)

Actually, I just found an instance at the top of 43v where a _porq'_ runs across a line, and it looks like Ximenez hyphens it to show it's connected. I wonder if that is applicable to the other instances of _porq'_?

**Comment by allisonmargaret** [comment]
I think it's okay to treat porq' the same as por q'. Colonial-era authors played fast and loose with spelling and spacing, and the copyist's hand was probably tired.

(Is that what you're asking, Aldo?)

Allison Margaret Bigelow
433 New Cabell Hall / PO Box 400777
Department of Spanish, Italian, and Portuguese
University of Virginia
Charlottesville, VA 22904
amb8fk@virginia.edu * +1.434.243.3371 (p)
http://spanitalport.as.virginia.edu/people/profile/amb8fk
________________________________

**Comment by aldobarriente** [comment]
Yes. I just wasn't sure if we were treating _porq'_ or _por q'_ as similar because of their linguistic function as "because" or as separate because we want to preserve the space that Ximenez has in some instances and not in others.

**Comment by aldobarriente** [comment]
Another question: should the expansions of _qo_ (corresponding to _cuando_) be expanded as modern orthography (_cuando_) or maintain the manuscript's orthography (_quando_)?

**Comment by allisonmargaret** [comment]
I’ve been writing quando on my branch. The commits are waiting in the cue.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by aldobarriente** [comment]
My most recent pull request added and fixed as many expansions as I could find and rechecked from top to bottom. Once you submit your pull request we can resolve the issue!

**Comment by allisonmargaret** [comment]
When I submit a new pull request, I get this message: "Multepal:master is up to date with all commits from allisonmargaret:master."

Allison Margaret Bigelow
433 New Cabell Hall / PO Box 400777
Department of Spanish, Italian, and Portuguese
University of Virginia
Charlottesville, VA 22904
amb8fk@virginia.edu * +1.434.243.3371 (p)
http://spanitalport.as.virginia.edu/people/profile/amb8fk
________________________________
